packages/fxa-content-server/app/scripts/views/settings/subscription.js

```diff
@@ -26,6 +25,36 @@ const View = FormView.extend({
     }
   },
 
+  template() {
+    return '';
+  },
+
+  afterRender() {
+    return this._hasActiveSubscriptions().then(hasSubs => {
+      if (hasSubs) {
+        this.$el.html(this.renderTemplate(Template));
+      }
+    });
+  },
+
+  /**
+   * Check to see if the account has any active subscriptions.
+   *
+   * @returns {Promise} resolves to a bool.
+   */
+  _hasActiveSubscriptions() {
+    return new Promise((resolve, reject) => {
```

Comment:
I don't think you need to wrap the Promise code here. Unless I'm missing something, this should be enough:
```javascript
_hasActiveSubscriptions() {
  return this.getSignedInAccount()
    .fetchActiveSubscriptions()
    .then(subscriptions => subscriptions.length > 0);
}
```

Reply:
Also, it might be a good idea someday to use the new account data API resource. But, we can do that as a followup and not block this PR.

Reply:
Updated the promise code.  Thanks!

I was not aware of the new account data API.  Where should I look?

Reply:
Filed https://github.com/mozilla/fxa/issues/1921